### DISCUSSION TITLE
Does this app run in the background?

### DISCUSSION THREAD

**Discussion by xVemu** [valinet/ExplorerPatcher#2200] [Q&A] [answered]
Does this app work in background and do some things like ex. TranslucentTB, or it uses some tricks (changes in registry etc.) to change windows behavior?

**Comment by pyrates999** [comment]
It is a combination of registry changes for some settings, and for others, it patches executables and libraries in memory using debug symbols.

**Comment by Amrsatrio** [comment, marked as answer]
This software is in form of a DLL gets loaded into Explorer (and, on your consent, programs that you run) so technically yes.
